src/views/contact-us/l10n.json

```diff
@@ -0,0 +1,16 @@
+{
+    "contactUs.title":"Contact Us",
+    "contactUs.intro":"You can find answers to most questions about Scratch on our {faqLink} page.",
+    "contactUs.faqLinkText":"Frequently Asked Questions",
+    "contactUs.forumsInfo":"You can also look through the Scratch {forumsLink}.",
+    "contactUs.forumsLinkText":"Discussion forums",
+    "contactUs.forumsAsk":"If you can't find an answer, post a new topic on the {forumsLink} (see links below). There are many experienced Scratchers who can help with the topics below and more:",
+    "contactUs.questionsForum":"Questions About Scratch",
```

Comment:
Those forum names should not be translated because the forum itself isn't. Rather it helps to use customized URL for each language (+ translations).

Reply:
Another good point, but this matches the current functionality so I'll keep it as is. I'll add some context in transifex for this version and it's something we can thing about for the future.